package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "setup:docs": "./scripts/download-dependencies.sh",
     "setup:native": "./scripts/download-dependencies.sh && ./scripts/setup-react-android.sh",
-    "postinstall": "yarn-deduplicate && expo-yarn-workspaces check-workspace-dependencies && ([ ! -f node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt.orig ] && patch -d node_modules -p2 -f --backup < patches/@shopify+flash-list+1.4.3.patch || true) && yarn workspace @expo/cli prepare",
+    "postinstall": "yarn-deduplicate && expo-yarn-workspaces check-workspace-dependencies && ([ ! -f node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt.orig ] && patch -d node_modules -p2 -f --backup < patches/@shopify+flash-list+1.4.3.patch || true) && ([ ! -f node_modules/@react-native/dev-middleware/package.json.orig ] && patch -d node_modules -p2 -f --backup < patches/@react-native+dev-middleware+0.73.4.patch || true) && yarn workspace @expo/cli prepare",
```

Comment:
Temporary patch while https://github.com/facebook/react-native/pull/41370 is not released